english/learn-climbing/learn-climbing.md

```diff
@@ -0,0 +1 @@
+"You know nothing about climbing!", said master Vaerys."But do not worry. YOu will learn it with patience and perseverance. Just like you made your first steps in github, before you will now be released into the online world full of repositories."
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.